### PR TITLE
Cryptonet testnet config

### PR DESCRIPTION
**Pull request by cryptonetD**
Added config for lumenx testnet

### CODE REVIEW

File: testnets/lumenxtestnet/chain.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "lumenx",
```
Comment (JeremyParish69) [inline testnets/lumenxtestnet/chain.json:3]: this chain_name must be 'lumenxtestnet'

File: testnets/lumenxtestnet/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+    "$schema":"../assetlist.schema.json",
+    "chain_name":"lumenx",
```
Comment (JeremyParish69) [inline testnets/lumenxtestnet/assetlist.json:3]: this chain_name must be 'lumenxtestnet'

Comment (cryptonetD) [reply testnets/lumenxtestnet/assetlist.json:3]: Ok, I will repair it in a minute

File: testnets/lumenxtestnet/chain.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "lumenxtestnet",
+  "status": "live",
+  "network_type": "mainnet",
```
Comment (JeremyParish69) [inline testnets/lumenxtestnet/chain.json:5]: oh, and this should be 'testnet'

File: testnets/lumenxtestnet/assetlist.json (hunk 2 above)
Comment (JeremyParish69) [reply testnets/lumenxtestnet/assetlist.json:3]: I see you updated the chian.json's chain_name, but I think you missed this assetlist file's chain_name property, which also must be 'lumenxtestnet'.

**Comment by cryptonetD** [comment]
Is there any other thing to be changed in the code?

**Comment by JeremyParish69** [comment]
> Is there any other thing to be changed in the code?

Yes, the chain_name in the assetlist.json file must be 'lumenxtestnet'

File: testnets/lumenxtestnet/assetlist.json (hunk 2 above)
Comment (cryptonetD) [reply testnets/lumenxtestnet/assetlist.json]: Yes, unfortunatelly, sorry for the inconvenience.

**Pull request merged**